README.md

```diff
@@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
```

Comment:
Or help give graphics for a slide deck...
On Sep 9, 2015 19:42, "W. Trevor King" <notifications@github.com> wrote:

> In README.md
> <https://github.com/opencontainers/specs/pull/168#discussion_r39110896>:
>
> > @@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
> >
> >  You can add the sign off when creating the git commit via `git commit -s`.
> >
> > +### Commit Style
> > +
> > +Simple house-keeping for clean git history.
> > +
> > +1. Separate subject from body with a blank line
> > +2. Limit the subject line to 50 characters
>
> On Wed, Sep 09, 2015 at 04:38:49PM -0700, Vincent Batts wrote: > +2. Limit
> the subject line to 50 characters Jesus Trevor... I ....
> Graphs help people make decisions, right? ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/168/files#r39110896>.
>